x-pack/plugins/apm/server/routes/errors/queries.test.ts

```diff
@@ -36,6 +37,22 @@ describe('error queries', () => {
     expect(mock.params).toMatchSnapshot();
   });
 
+  it('fetches the error sample by errorId', async () => {
+    mock = await inspectSearchParams(({ mockApmEventClient }) =>
+      getErrorGroupSampleDetails({
+        errorId: 'errorId',
+        serviceName: 'serviceName',
+        apmEventClient: mockApmEventClient,
+        environment: ENVIRONMENT_ALL.value,
+        kuery: '',
+        start: 0,
+        end: 50000,
+      })
+    );
+
+    expect(mock.params).toMatchSnapshot();
+  });
+
   it('fetches multiple error groups', async () => {
     mock = await inspectSearchParams(({ mockApmEventClient }) =>
       getErrorGroupMainStatistics({
```

Comment:
I suggest deleting this entire file an instead making sure we've got this covered by API tests. Too much mocking going on that can be replaced with the actual implementation.